src/test/java/org/jabref/model/entry/AuthorListTest.java

```diff
@@ -20,7 +20,7 @@ public void testFixAuthorNatbib() {
         // Is not cached!
         Assert.assertTrue(AuthorList
                 .fixAuthorNatbib("John von Neumann and John Smith and Black Brown, Peter").equals(AuthorList
-                        .fixAuthorNatbib("John von Neumann" + (0 == 1 ? "" : " and ")
+                        .fixAuthorNatbib("John von Neumann" + (false ? "" : " and ")
```

Comment:
Why not removing the condition completly? Just `"John von Neumann and John Smith and Black Brown, Peter"`?

Similar comments for the other `false` checks.